local/php/toolbar.go

```diff
@@ -81,23 +82,32 @@ func (p *Server) tweakToolbar(body io.ReadCloser, env map[string]string) (io.Rea
 		}
 	}
 
-	webmail := `<span class="sf-toolbar-status sf-toolbar-status-red">Down</span>`
-	rabbitmqui := `<span class="sf-toolbar-status sf-toolbar-status-red">Down</span>`
+	webmail := `<b>Webmail</b> <span class="sf-toolbar-status sf-toolbar-status-red">Down</span>`
+	rabbitmqui := `<b>RabbitMQ UI</b> <span class="sf-toolbar-status sf-toolbar-status-red">Down</span>`
 	blackfire := `<span class="sf-toolbar-status sf-toolbar-status-red">Down</span>`
+	extraLinks := ``
 	if env, err := envs.NewLocal(p.projectDir, terminal.IsDebug()); err == nil {
-		values := envs.AsMap(env)
-		if prefix := env.FindRelationshipPrefix("mailer", "http"); prefix != "" {
-			if url, exists := values[prefix+"URL"]; exists {
-				webmail = fmt.Sprintf(`<span class="sf-toolbar-status sf-toolbar-status-green">Up</span>&nbsp;&nbsp;&nbsp;<a class="sf-cli-webmail" href="%s" target="_blank">Open</a>`, url)
-			}
+		if url, exists := env.FindServiceUrl("mailer"); exists {
+			webmail = fmt.Sprintf(`<b><a class="sf-cli-webmail" href="%s" target="_blank">Webmail</a></b> <span class="sf-toolbar-status sf-toolbar-status-green">Up</span></a>`, url)
```

Comment:
For security reasons, we could add `rel="noopener"` to all `target="_blank"` links. Some browsers like Chrome do that automatically, but others don't do that yet.

See https://developer.chrome.com/docs/lighthouse/best-practices/external-anchors-use-rel-noopener/

Reply:
Works for me.

Reply:
makes sense 👍 
done